### ISSUE TITLE
Custom permissions not working

### ISSUE THREAD

**Issue by raeesaroj** [django-mptt/django-mptt#753]
class MPTTMeta:
        order_insertion_by = ['created_date']
        db_table = 'project_table'
        permissions = (
            ("custom_perm_1", "Create custom permission 1"),
            ("custom_perm_2", "Create custom permission 2"),
        )

Migration is not working so the above two custom permissions are not found in the table.

**Comment by matthiask** [comment]
Yes, `db_table` and `permissions` belong into a `class Meta` class. (`order_insertion_by` is correct inside `class MPTTMeta`.)

**Comment by raeesaroj** [comment]
> Yes, `db_table` and `permissions` belong into a `class Meta` class. (`order_insertion_by` is correct inside `class MPTTMeta`.)

So I have to create custom permissions programmatically. right?

**Comment by matthiask** [comment]
See #833 — the project is in maintenance-only mode. For usage questions like this one, Stack Overflow or the Django Forum are usually better venues. Thanks for your understanding.
